recipe/meta.yaml

```diff
@@ -18,14 +18,18 @@ package:
   version: {{ version }}
 
 source:
+  {% if platform_segment is defined %}
   url: {{ root_url }}/{{ platform_segment }}/packages/{{ name }}.{{ file_ext }}
+  {% endif %}
+  {% if sha256 is defined %}
   sha256: {{ sha256 }}
+  {% endif %}
```

Comment:
Are you opening this PR to fix a specific bug somewhere or just from the comments left here: https://github.com/conda-forge/cspice-feedstock/pull/40#discussion_r1857407909

If you revert this change the linter would pass?

Reply:
these were just the comments from #40 that got lost when I closed that pr (I may have accidentally deleted the branch)

Reply:
Looks like builds fail otherwise? See #43